conans/test/integration/graph/test_dependencies_visit.py

```diff
@@ -29,6 +29,12 @@ def generate(self):
                 self.output.info("DefPRefBuild: {}!!!".format(dep.pref.repr_notime()))
                 for r, d in self.dependencies.build.items():
                     self.output.info("DIRECTBUILD {}: {}".format(r.direct, d))
+
+                if "openssl" in self.dependencies:
```

Comment:
It would be good to extend the testing a little bit and cover the ``in`` cases for ``tool_requires``, both in ``self.dependencies`` and ``self.dependencies.build``

Reply:
Done, please re-check :)